neuralop/layers/codano.py

```diff
@@ -0,0 +1,462 @@
+from functools import partial
+import logging
+import numpy as np
+import torch
+from einops import rearrange
+import torch
+from torch import nn
+import torch.nn.functional as F
+from .fno_block import FNOBlocks
+from .spectral_convolution import SpectralConv1d, SpectralConv2d, SpectralConv3d
+
+
+AffineNormalizer2D = partial(nn.InstanceNorm2d, affine=True)
+AffineNormalizer3D = partial(nn.InstanceNorm3d, affine=True)
```

Comment:
Is it worth defining this? I'd just explicitly pass the affine keyword

Reply:
I have removed these; however, "affine" cannot be passed as a keyword because we are using "Normalizer," which is callable and may not have the affine keyword if it is some custom normalizer function.